docker.go

```diff
@@ -37,18 +37,20 @@ type (
 
 	// Build defines Docker build parameters.
 	Build struct {
-		Remote      string   // Git remote URL
-		Name        string   // Docker build using default named tag
-		Dockerfile  string   // Docker build Dockerfile
-		Context     string   // Docker build context
-		Tags        []string // Docker build tags
-		Args        []string // Docker build args
-		ArgsEnv     []string // Docker build args from env
-		Squash      bool     // Docker build squash
-		Pull        bool     // Docker build pull
-		Compress    bool     // Docker build compress
-		Repo        string   // Docker build repository
-		LabelSchema []string // Label schema map
+		Remote        string   // Git remote URL
+		Name          string   // Docker build using default named tag
+		Dockerfile    string   // Docker build Dockerfile
+		Context       string   // Docker build context
+		Tags          []string // Docker build tags
+		Args          []string // Docker build args
+		ArgsEnv       []string // Docker build args from env
+		Squash        bool     // Docker build squash
+		Pull          bool     // Docker build pull
+		Compress      bool     // Docker build compress
+		Repo          string   // Docker build repository
+		LabelSchema   []string // Label schema map
+		Branch        string   // Docker build branch
+		DefaultBranch string   // Docker latest branch
```

Comment:
I think this can be removed since it isn't being used

Reply:
Done. https://github.com/drone-plugins/drone-docker/pull/160/commits/c591da7e86eb36cded8d7839382d312350b87676